res/drawable/issue_event_closed.xml

```diff
@@ -0,0 +1,15 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillAlpha="1"
+        android:fillColor="#e65858"
```

Comment:
I never got to look into compat vector drawables so far. If one can use those in menus etc., they sound like an even better choice, because one then could use theme attributes for the colors and get rid of the drawable duplication.